### DISCUSSION TITLE
Question about subtitles timing

### DISCUSSION THREAD

**Discussion by AndrewHoover** [mifi/lossless-cut#1516] [Q&A]
I am trying to cut about 20-30 seconds from the beginning of a mp4 video that contains a subtitle track. However, the rendered video has the subtitle track still offset by the original longer video length. Is there a way to sync up the subtitle track with the new shorter video?

**Comment by mifi** [comment]
ffmpeg sometimes has problems with subtitles. Not sure how to fix that unfortunately. if you can reproduce the issue with an ffmpeg command you could try to report an issue over at ffmpeg
